src/test/java/picard/util/LiftoverUtilsTest.java

```diff
@@ -0,0 +1,98 @@
+package picard.util;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.variantcontext.VariantContextComparator;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import picard.vcf.VcfTestUtils;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by farjoun on 1/16/18.
+ */
+public class LiftoverUtilsTest {
+
+    static final Allele A = Allele.create("A", false);
+    static final Allele T = Allele.create("T", false);
+    static final Allele C = Allele.create("C", false);
+    static final Allele RefT = Allele.create("T", true);
+    static final Allele RefA = Allele.create("A", true);
+    static final Allele RefC = Allele.create("C", true);
+    static final Allele RefG = Allele.create("G", true);
+
+    static String AF = "AF", MAX_AF = "MAX_AF";
```

Comment:
I prefer each declaration on its own line.  Not sure if we have a standard for this.